components/alert/src/Alert.js

```diff
@@ -0,0 +1,146 @@
+/* @jsx jsx */
+
+import React, { useState } from 'react';
+import { css, jsx, useTheme } from '@westpac/core';
+import Color from 'color';
+
+import { CloseIcon } from '../../icon/src'; //until icon package is published
+import { CSSTransition } from 'react-transition-group';
+
+// ==============================
+// Utils
+// ==============================
+
+// ==============================
+// Component
+// ==============================
+const styleAppearance = appearance => {
+	const theme = useTheme();
+	return css({
+		color: theme.colors[appearance],
+		display: 'flex',
+		alignItems: 'center',
+
+		'& > .icon': {
+			marginRight: '6px',
+			marginTop: '1px',
+			float: 'left',
+		},
+
+		'& > .icon-alert': {
+			marginRight: '4px',
+		},
+
+		a: {
+			color: 'inherit',
+		},
+	});
+};
+
+const styleBox = appearance => {
+	const theme = useTheme();
+	return css({
```

Comment:
Lets not use Emotion's css function like this; return styling in simple object notation

```
return {
   ...
};
```